from .interruption import assert_continue, InterruptException